core/src/main/java/me/deadlight/ezchestshop/data/Config.java

```diff
@@ -99,13 +99,19 @@ public class Config {
 
     public static boolean shopCommandsEnabled;
     public static ShopCommandManager shopCommandManager;
+    public static int taxesshopplayer;
+    public static int taxesplayershop;
+    public static String leader;
 
 
     public static void loadConfig() {
         //reloading config.yml
 
         EzChestShop.getPlugin().reloadConfig();
         FileConfiguration config = EzChestShop.getPlugin().getConfig();
+        taxesshopplayer = config.getInt("economy.taxes-shop-player");
```

Comment:
While I get that this means buying/selling taxes, I think it would be less confusing if it would say something like
- `economy.taxes.sell-to-shop-tax-percent`
- `economy.taxes.buy-from-shop-tax-percent`